src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -12,30 +12,32 @@ This feature is available only if your organization is on either a Business or T
 
 </Note>
 
-Custom Dashboards are shared across your organization, and users can view each other’s custom dashboards by selecting them from the dropdown menu in the top right. Note that any changes you make in the [default dashboard](/product/dashboards/) (called Dashboard) are global, and will appear for users across the entire organization.
+Custom Dashboards are shared across your organization, and users can view each other’s custom dashboards in the management view. Note that any changes you make are global, and will appear for users across the entire organization.
 
-![Dashboard navigation.](dashboard_cropped.png)
+## Manage Dashboards
+When you have either a Business or Trial Plan, you'll land on a view to manage dashboards instead of the [default dashboard](/product/dashboards/). From here, you can:
```

Comment:
```suggestion
When you have either a Business or Trial Plan, navigating to **Dashboards** takes you to a view where you can manage dashboards instead of the [default dashboard](/product/dashboards/). From here, you can:
```